resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh

```diff
@@ -17,13 +17,15 @@
 #
 
 # echo commands to the terminal output
-set -ex
+set -x
 
 # Check whether there is a passwd entry for the container UID
 myuid=$(id -u)
 mygid=$(id -g)
 uidentry=$(getent passwd $myuid)
 
+set -e
```

Comment:
it is probably best to limit the "no -e" specifically to `getent`, such as:
```bash
set -ex
myuid=$(id -u)
mygid=$(id -g)
# turn off -e for getent because it will return error code in anonymous uid case
set +e
uidentry=$(getent passwd $myuid)
set -e
```